doc/source/reporting.rst

```diff
@@ -138,7 +140,7 @@ Others:
 
    Some notes:
 
-   - A Key has the same hash, and compares equal to its ``str()``.
+   - A Key has the same hash, and compares equal (NOT CLEAR THIS SENTENCE) to its ``str()``.
```

Comment:
I don't understand what you meant in this sentence, @khaeru . I left that comment there, pending to be removed in the next commit.

Reply:
"compares equal" means using the `==` operator in Python. Sometimes `==` is picky, e.g.:
```
>>> 1 == '1'
False
```
This happens because `1` is an int, and `'1'` is a string.

If you have a Key object on one side of `==`, and a string object on the other side, it will return True. That's showed further up on lines 125–127:
```
>>> k1 = Key('foo', ['a', 'b', 'c'])
>>> k1 == 'foo:a-b-c'
True
```
Here, `k1` is the Key object, and `'foo:a-b-c'` is a string object.

This functionality is provided by `Key.__eq__`: https://github.com/iiasa/ixmp/blob/480d0d81e74e303dd2e3ef1aed3007abbfc3618a/ixmp/reporting/key.py#L93-L95

With this explanation, I'd welcome your suggestion on how to rephrase to make the documentation clearer.

Reply:
Thank you 😃